README.md

```diff
@@ -1,12 +1,75 @@
 celvet
 ======
 
+**celvet** is a multitool for linting CRDs (`CustomResourceDefinition`) in
```

Comment:
What does multitool mean?

Reply:
[This](https://en.wikipedia.org/wiki/Multi-tool) is what I was going for (to get the idea across that celvet can/will be able to solve a lot of different use cases), but maybe a different introductory sentence would be better.